docker/docker-compose-deployment/initialize-databases-if-necessary.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd "${DOCKER_COMPOSE_DEPLOYMENT_DIR}" || exit 1
+. .env
+
+printf "\nStarting the database initializing procedure.\n"
+
+echo "  Starting database."
+docker run --rm -d \
+  --name "${DATABASE_CONTAINER_NAME}" \
+  -v "${DATABASE_VOLUME}":/var/lib/postgresql/data \
+  -e POSTGRES_PASSWORD="${POSTGRES_PASSWORD}" \
+  postgres:"${POSTGRES_VERSION}"
+
+echo "  Waiting a few seconds until database is ready."
+sleep 5
```

Comment:
I also found this article interesting: https://docs.docker.com/compose/startup-order/

Best practice would be to implement logic in our application to wait. Probably this can be configured quite easily by setting the correct configuration for hibernate in the application.yml.

Reply:
Could be moved to a follow-up issue.